internal/command/views/json/change_summary.go

```diff
@@ -21,12 +22,29 @@ type ChangeSummary struct {
 // used by Terraform Cloud and Terraform Enterprise, so the exact formats of
 // these strings are important.
 func (cs *ChangeSummary) String() string {
+
+	// TODO(liamcervante): For now, we only include the import count in the plan
+	//   output. This is because counting the imports during the apply is tricky
```

Comment:
I know this is only a TODO, but I'm not sure if the hooks should get called during apply, because the entirety of the import process happens during planning. For example, if you were to apply and there was an error early on, all of the "imported" instances would still have been imported even if their plans were not applied. I wonder if we can generate the counts directly from the changes or not.

Reply:
however you want to handle it! Just trying to save you from going down a path which may not work 😉

Reply:
I've taken any mention of how we will implement this for the apply, just left in a statement that we will.